test/unit/project_type/test_git.py

```diff
@@ -175,6 +179,14 @@ def test_slave_param_overrides_returns_expected(self):
         self.assertEqual(expected_overrides, actual_overrides, 'Slave param overrides from Git object should match'
                                                                'expected.')
 
+    def test_slave_param_overrides_returns_empty_dict_when_get_project_from_master_is_disabled(self):
```

Comment:
What do you think of just asserting that 'url' and 'branch' are not keys in the overrides dict? It might make this test more stable against future unrelated additions to slave param overrides.